src/data/maps.js

```diff
@@ -65,6 +66,7 @@ export const cropIdToTypeMap = {
   [SOYBEAN]: 'soybean',
   [SPINACH]: 'spinach',
   [STRAWBERRY]: 'strawberry',
+  [SWEET_POTATO]: 'sweet-potato',
```

Comment:
This is a total aside not at all related to this PR (and not a request for a change), but I'd love to redesign the crop definition system a bit so there aren't so many touch points for adding crops. It's kind of laborious to add new crops, and I think we could have something a little more streamlined.